client/src/main.jsx

```diff
@@ -2,9 +2,18 @@ import { StrictMode } from "react";
 import { createRoot } from "react-dom/client";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
 import "bootstrap/dist/css/bootstrap.css";
+import { Auth0Provider } from "@auth0/auth0-react";
 import App from "./App";
 import EditorTab from "./EditorTab";
 import HomePage from "./HomePage";
+
+const domain = import.meta.env.VITE_AUTH_DOMAIN;
+const clientId = import.meta.env.VITE_AUTH_CLIENT_ID;
```

Comment:
@Yash-Shukla-9 Can we move these variables to some other file to maintain data abstractions